login-workflow/src/components/Spinner.tsx

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
-import { CircularProgress, makeStyles, createStyles } from '@material-ui/core';
+import { CircularProgress, makeStyles, createStyles, useTheme, Theme } from '@material-ui/core';
 
-const useStyles = makeStyles(() =>
+const useStyles = makeStyles((theme: Theme) =>
     createStyles({
         background: {
-            position: 'absolute',
+            position: 'fixed',
```

Comment:
So this change from absolute to fixed is what caused the scrim to go over the entire page instead of only the card. Is this desired @huayunh ?

Reply:
Ahh, that makes sense. I ended up using that to fix the scrim not covering the entire card on on a few screens at mobile viewports. I didn't realize the side effects until now; I can try to find another workaround for this.

Reply:
Yeah, if you can find another way of doing this I think that would be better...it feels a little weird to have the scrim over the entire page.

Reply:
Turns out this is a big pain in the butt...I'm fine with using position fixed for now.